components/ei-analytics/org.wso2.analytics.solutions.ei.analytics/widgets/EIAnalyticsTPS/src/EIAnalyticsTPS.jsx

```diff
@@ -132,7 +131,7 @@ class EIAnalyticsTPS extends Widget {
                     );
             })
             .catch((error) => {
-                // console.log(error);
+                console.error("Unable to load widget configurations");
```

Comment:
Of what widget? Error messages should be context ware, meaningful, and descriptive.
e.g. 
```js
`Cannot load configurations of '${this.props.widgetID}' widget.
```

Reply:
Applies for other places as well.

Reply:
Fixed at 8d8a3594932df5bd7844ec4ff189f485c7408493